collect_app/src/main/java/org/odk/collect/android/external/ExternalSQLiteOpenHelper.java

```diff
@@ -263,4 +265,23 @@ private void onProgress(String message) {
             formLoaderTask.publishExternalDataLoadingProgress(message);
         }
     }
+
+    /**
+     * replaceByteOrderMark
+     *
+     * Removes the presence of a Byte Order Mark from the initial UTF-8 String
+     *
+     * Fixes issue: https://github.com/opendatakit/collect/issues/1695
+     *
+     * @param firstLine String to scan for a Byte Order Marker
+     * @return String with any Byte Order Marker removed
+     */
+    private String replaceByteOrderMark(String firstLine) {
+        final String BOM = "\uFEFF";
+        if (firstLine.startsWith(BOM)) {
```

Comment:
Could you remove this if else statement using just:
`return firstLine.startsWith(BOM) ? firstLine.substring(1) : firstLine`

Reply:
It's not a must but if you look at our code (newly implemented changes) this approach is popular and other reviewers also prefer the ternary operator like me.

Reply:
Ok, fair enough. I'll make the change to be consistent with other code.

Thanks